packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Removing stack trace for zosjobs errors. [#2078](https://github.com/zowe/zowe-cli/pull/2078)
```

Comment:
For the changelog, do we want to mention something about fixing prompting behavior for user/password on SSH commands? Will defer to others reviewing this PR 😋